example/web-node-sample/web-client/index.html

```diff
@@ -0,0 +1,178 @@
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <script src="https://cdn.jsdelivr.net/npm/axios/dist/axios.min.js"></script>
+  <link href="style.css" rel="stylesheet" type="text/css">
+  <script
+      src="../scalardl-web-client-sdk.bundle.js"></script>
+</head>
+
+<body>
+<div id="main-container">
+  <img src="../img/scalar_logo_horizontal.jpg" width="100%">
+  <h2 style="color: rgb(60, 103, 177)">Sample Code demonstration</h2>
+
+  <div class="Component">
+    <button id="registerCertificateBtn">
+      Register Certificate
+    </button>
+    <pre><code>
+await clientService.registerCertificate();</code></pre>
+    <br>
+  </div>
+  <pre><code id="registerCertificateResult"></code></pre>
+  <br>
+  <br>
+  <div style="color: rgb(90, 90, 90)">
+    <b>Contract:</b> <input id="contractBinary" type="file"/>
+  </div>
+  <div class="Component">
+    <button id="registerContractsBtn">
+      Register Contracts
+    </button>
+    <pre><code>
+await clientService.registerContracts(contractId, contractName, contractBinary, {});</code></pre>
+  </div>
+  <pre><code id="registerContractResult"></code></pre>
+  <br>
```

Comment:
Actually, i was abit of a trouble for me to do that, because on the browser, we can't simply read local file, i need to investigate on how to overcome this...[here](https://developer.mozilla.org/en-US/docs/Web/API/FileReader) it says can be achieved using Ajax...

Reply:
Ok, if it's really troublesome let's think about something else

Reply:
How about i put the StateUpdater.class file same directory as the html...

Reply:
Ok, let's do this